infrastructure/cluster/flux-v2/jhub/jhub-release.yaml

```diff
@@ -162,10 +168,20 @@ spec:
           - name: eulake-escape-tmpdata # mounts the EOS RSE needed for the Rucio JupiterLab extension
             mountPath: /eos/scratch-space
             mountPropagation: HostToContainer
+          - name: eulake-escape-data-test-fusemount
```

Comment:
You need to use the exact/correct name. Otherwise, the volume will not be found.

Reply:
Right, thanks !